package.json

```diff
@@ -2,13 +2,14 @@
   "name": "nextjs-commerce",
   "version": "1.0.0",
   "scripts": {
-    "dev": "next dev",
+    "dev": "next dev -p 8080",
     "build": "next build",
     "start": "next start",
     "analyze": "BUNDLE_ANALYZE=both yarn build",
     "prettier-fix": "prettier --write .",
     "find:unused": "next-unused",
     "generate": "graphql-codegen",
+    "generate:reactioncommerce": "graphql-codegen --config framework/reactioncommerce/codegen.json",
```

Comment:
@lfades  I wonder if it's better to add a .graphqlrc, or .graphqlrc.yml/yaml instead. wdyt?
Also, the buildsteps can be in the Vercel project.

Reply:
Discussed with @lfades. We're good for now 👍